subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -1,6 +1,7 @@
 /*
```

Comment:
may be create new PR for legacy controller changes?

Reply:
I can take the relevant commit out of this PR and add it to a new one, but I'd rather have everything Vega SW LL related into one PR. There's also the option of dropping the legacy support altogether. What do you think?

Reply:
If the legacy controller is working then I have no objection to keep it. If its not too much work to keep it in separate commit, please do. It will be easy in future in case doing a bisect or to revert just legacy controller file changes.

Reply:
@alexandru-porosanu-nxp Given that the legacy controller is meant to be deprecated in around 1 release, it may be just simpler to drop its changes, and therefore minimize the amount of extra work.

Reply:
OK then, I will drop the corresponding commit from the PR

Reply:
My suggestion was to remove legacy controller changes for the exact same reason @aescolar made. To me it simplifies the code to remove the unnecessary conditional compile blocks.